### ISSUE TITLE
Unable to build with mkdocs 1.4.2

### ISSUE THREAD

**Issue by jachstet-sea** [smarie/develop-behind-proxy#5]
Since we got changes with #4, I tried to build & deploy this. I installed mkdocs-1.4.2 and mkdocs-material-9.1.5. However, the `mkdocs build` step fails with:
```
jachstet@GentooVM ~/git/develop-behind-proxy $ mkdocs build
ERROR    -  Config value 'pages': The configuration option 'pages' was removed from MkDocs. Use 'nav' instead.
Aborted with 1 Configuration Errors!
```

Either we need to document the version requirements or we need to update the configuration to work with the latest versions.

**Comment by jachstet-sea** [comment]
Fixed by e9b7fb0

**Comment by smarie** [comment]
Great @jachstet-sea , 
indeed this project has not been modified for a long time now so the mkdocs config was obsolete
thanks for fixing this!
